examples/flows/chat/chat-with-pdf/chat-with-pdf.ipynb

```diff
@@ -130,7 +130,7 @@
     "\n",
     "config_2k_context = {\n",
     "    \"EMBEDDING_MODEL_DEPLOYMENT_NAME\": \"text-embedding-ada-002\",\n",
-    "    \"CHAT_MODEL_DEPLOYMENT_NAME\": \"gpt-35-turbo\",\n",
+    "    \"CHAT_MODEL_DEPLOYMENT_NAME\": \"gpt-4\", # change this to the name of your deployment if you're using Azure OpenAI\n",
```

Comment:
should we leave deployment_name empty in all samples? receive lots of feedback from AOAI users that the first run failed due to deployment not available error, as they were not aware of the deployment name need to be modified according to their own AOAI resource.

Reply:
In this specific example, the CHAT_MODEL_DEPLOYMENT_NAME is shared between open ai and azure open ai ...